pkg/cloud/openstack/machine/actuator.go

```diff
@@ -266,6 +270,13 @@ func (a *Actuator) Exists(ctx context.Context, cluster *clusterv1.Cluster, machi
 	if err != nil {
 		return false, err
 	}
+
```

Comment:
Not sure we want this as a side effect of the exists method.

Reply:
I referred to AWS previous implementation, seems under current arch there is no good way to set value except in this function...